lib/elixir/lib/calendar/date.ex

```diff
@@ -942,7 +942,7 @@ defmodule Date do
         %Date{calendar: calendar, year: year, month: month, day: day}
 
       {day_of_week, first_day_of_week, _} ->
```

Comment:
Change to:
```elixir
{day_of_week, first_day_of_week, last_day_of_week}
```
so that we can use `max(first_day_of_week, last_day_of_week)` later on.